feat: Add copy button for both login & display name inside the user info popout

1xelerate opened the pull request:
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Create a new icon for the new copy button? (help)

# Description

![image](https://user-images.githubusercontent.com/59930937/139605521-bb8f980a-516c-46f1-a561-8ea2e78211a5.png)


Make the copy-text of the original button match the name shown, which is the display name of the user. Additionally, added a new button next to the original one that copies the login name of the user instead of the localized display name version shown in the popout. 

Currently both buttons have the same icon, which doesn't look that bad imo but pajlada had mentioned probably wanting a different icon. I am quiet clueless when it comes to anything art related and couldn't think of anything good for the new button, nor knew how to properly make the svg's. I am hoping someone might have a good idea?

Closes #3070

1xelerate commented:
Proposal for new icon(s) for both the normal copy button and the new login copy button. I increased the size of the buttons slightly and made the dim less intense for clarity. I feel like im toeing the line of going out of scope of this PR but let me know what you think!

![image](https://user-images.githubusercontent.com/59930937/139612992-490a47bd-07db-450d-893d-06de66ed636d.png)
![image](https://user-images.githubusercontent.com/59930937/139613014-eddc678d-6794-4694-affa-dd3e644e03af.png)

jammehcow commented:
This feels like the wrong solution to the problem (at least to me). Having two buttons differentiated by some inner text isn't instantly obvious to a user as to what they do. Even with a tooltip as suggested in the issue, that requires a user to hold the mouse over the button to see what action it does.

I feel like the best way to do this is one of the following:
 - Show the user's localized name with the copy button & the login name below with another copy button
 - Put the login name in parentheses next to the localized name with a copy button inside

Would've put this on the issue but you've created a PR now :)

NS2555 commented:
> I feel like the best way to do this is one of the following:
> 
> * Show the user's localized name with the copy button & the login name below with another copy button

i agree with this solution, i think it would be more intuitive

1xelerate commented:
> This feels like the wrong solution to the problem (at least to me). Having two buttons differentiated by some inner text isn't instantly obvious to a user as to what they do. Even with a tooltip as suggested in the issue, that requires a user to hold the mouse over the button to see what action it does.
> 
> I feel like the best way to do this is one of the following:
> 
>     * Show the user's localized name with the copy button & the login name below with another copy button
> 
>     * Put the login name in parentheses next to the localized name with a copy button inside
> 
> 
> Would've put this on the issue but you've created a PR now :)

Proposal 2: Reverted the new icons since I think it might be out of scope of this PR/Issue, and added an additional login name label next to the display name label if the user's login name is different than their display name (localized names).

![image](https://user-images.githubusercontent.com/59930937/139728060-2d5ada24-8d85-4d50-882f-763750cfc749.png)
![image](https://user-images.githubusercontent.com/59930937/139728067-7df1c34d-8da7-4881-b372-13c7de32ac6a.png)

pajlada commented:
Final look: 
![image](https://user-images.githubusercontent.com/962989/140609595-1197918b-af7f-41fc-90cd-9865d27a9cae.png)

Felanbird commented:
@1xelerate As I mentioned in chat my thoughts on standardizing the login name to the first position (since localized names are very strongly in the minority), I also noticed that the localized name would push the login name to the side when loading had finished, so I mentioned this point to pajlada and he agreed we should standardize it as seen by one of the above commits.
